bench.sh

```diff
@@ -265,6 +265,18 @@ function tests_run_add_remove { # <test> <add?>
   fi
 }
 
+function process_external { # file
+  REGEXP="^([^ ]+) +(.*)$"
+
+  while read -r ; do
+    echo $REPLY
```

Comment:
Remove this line, or even better, add some logging: `echo "Adding $REPLY"` in the `if`, and a `Not adding $REPLY because it's malformed` in an `else`.